index.js

```diff
@@ -1,18 +1,13 @@
-var Promise = typeof Promise === 'undefined' ? require('promise') : Promise,
-    promisify = require('promisify-node'),
-    globby = require('globby'),
-    path = require('path'),
-    eol = require('os').EOL,
-    globbyPromise = promisify(globby);
-
-function isLess(file) {
-    return path.extname(file) === '.less';
-}
+var Promise = require('promise');
+var globby = require('globby');
+var path = require('path');
+var eol = require('os').EOL;
+var globbyPromise = Promise.denodeify(globby);
 
 function processPaths(paths) {
-    return paths.filter(function(path) {
-        if(!isLess(path)) {
-            console.warn('Here is non-less file: ' + path + ', ignored');
+    return paths.filter(function(filepath) {
+        if (path.extname(filepath) !== '.less') {
```

Comment:
Why do you think that is nice to inline this function?
I considered to have a separate function with a proper description